waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -196,6 +199,29 @@ private void startWaggleDance(
     LOG.info("Waggle Dance has stopped");
   }
 
+  private TProcessorFactory getTProcessorFactory(boolean useSASL) throws TTransportException {
```

Comment:
Please rename to `createTProcessorFactory` (as it's not a getter, it's actually creating something)